### ISSUE TITLE
Work landing page QA

### ISSUE THREAD

**Issue by vickibrown** [thinkshout/thinkshout#825]
https://thinkshout.com/work/
- [x] Case study card text should turn hex F06D2F on hover/focus
- [x] Change client sector titles at the bottom to hex BF450D

**Comment by komejo** [comment]
Ready for review on https://a11y-2020--thinkshout.netlify.app/work/

**Comment by vickibrown** [comment]
@komejo looks good.

**Comment by JulesKhong** [comment]
- [ ] Update focus states on project cards to match hover states (orange outline with zoom)

**Comment by JulesKhong** [comment]
- [ ] Could we look at disconnecting styles from heading levels? They're directly tied to size, but not hierarchy currently. Client names should be H3's under the H2 "Work". "Select Clients" should be an H2, and client categories should each be an H3, so the content is organized clearly by heading. "New ideas about strategy, design, and tech for good." should be an H3 as well. @vickibrown Do you have input on headings as long as styles don't change? Should we grab a strategist to review?

**Comment by vickibrown** [comment]
@JulesKhong I don't care how things are labeled if the design doesn't change. 
@komejo For the button in the CTA, it looks like the color passes if it's larger text, so lets just make the button 18 pixels and bold so we don't have to redesign the entire CTA.

**Comment by komejo** [comment]
All feedback applied - this should be ready for final review/deploy: https://a11y-2020--thinkshout.netlify.app/work/
